src/style.css

```diff
@@ -1 +1,40 @@
 /* add styles here */
+body {
+  margin: 0 auto;
+  display: flex;
+}
+
+.main-block {
+  flex-direction: column;
+  display: flex;
+  margin: 0 auto;
+  height: 94px;
+  width: 100px;
```

Comment:
[Checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md): Don't fix container size. Let the content size dictate it.